src/state/item/item-reducers.js

```diff
@@ -13,6 +13,8 @@ const initialState = {
     activeItemIndex: null,
     activeItemIndexCached: null, // saves the index of active item on different page
     activeItemPage: null,
+    activeItemContent: null,
+    activeItemContentType: 'web', // TODO: figure out content type from response
```

Comment:
Check the response headers -- should be under `Content-Type`.  Also, I'd double check when you pull a pdf it doesn't attempt to download it in a way the user can interact with (this is what postman does for me).